src/components/custom-sliding-panel.vue

```diff
@@ -0,0 +1,41 @@
+<template>
```

Comment:
This component is used only in two places. And if these two places need a different configuration in the setup, then this is useless.

I think it is better to repeat in two places the configuration than to maintain this component in future upgrades.